### ISSUE TITLE
error on predict

### ISSUE THREAD

**Issue by Goghzki** [JaidedAI/EasyOCR#383]
![image](https://user-images.githubusercontent.com/22120157/109465699-8edc6b00-7aa3-11eb-8e29-aa0ac6ebb494.png)
how to avoid this error

**Comment by NyxCode** [comment]
just came here to report the same error.
Here is a minimal example to reproduce:
```python
import io
from easyocr import Reader
from PIL import Image

img = Image.open("temp.png")
buf = io.BytesIO()
img.save(buf, format="PNG")

reader = Reader(["ar"])
reader.readtext(buf.getvalue(), allowlist=["1234567890"], detail=0)
```
[This is "temp.png"](https://workupload.com/file/pE9JF38f4S9)
Full error:
```
Traceback (most recent call last):
  File "C:/Users/morit/Coding/bft/bug.py", line 10, in <module>
    reader.readtext(buf.getvalue(), allowlist=["1234567890"], detail=0)
  File "C:\Users\morit\Coding\bft\venv\lib\site-packages\easyocr\easyocr.py", line 391, in readtext
    result = self.recognize(img_cv_grey, horizontal_list, free_list,\
  File "C:\Users\morit\Coding\bft\venv\lib\site-packages\easyocr\easyocr.py", line 327, in recognize
    result0 = get_text(self.character, imgH, int(max_width), self.recognizer, self.converter, image_list,\
  File "C:\Users\morit\Coding\bft\venv\lib\site-packages\easyocr\recognition.py", line 206, in get_text
    result1 = recognizer_predict(recognizer, converter, test_loader,batch_max_length,\
  File "C:\Users\morit\Coding\bft\venv\lib\site-packages\easyocr\recognition.py", line 148, in recognizer_predict
    confidence_score = custom_mean(pred_max_prob)
  File "C:\Users\morit\Coding\bft\venv\lib\site-packages\easyocr\recognition.py", line 14, in custom_mean
    return x.prod()**(2.0/np.sqrt(len(x)))
AttributeError: 'list' object has no attribute 'prod'
```

**Comment by NyxCode** [comment]
Also, using JPEG (`img.save(buf, format="JPEG")`) seems to fix it, too..

**Comment by NyxCode** [comment]
pretty weird though, it works fine for other images. There's definetely something funky going on here

**Comment by poonyapat** [comment]
Same error, I found it's about the new 1.2.5 confident score so change version to 1.2.4 might help you.

**Comment by Goghzki** [comment]
> Also, using JPEG (`img.save(buf, format="JPEG")`) seems to fix it, too..

can I consider the reason is the PNG format image is too big?  I need to convert it to JPEG or set maximal pixel if I insist use PNG format?

**Comment by rkcosmos** [comment]
thanks everyone for this bug report, I'll investigate and fix soon.

**Comment by rkcosmos** [comment]
@Goghzki @NyxCode I'm unable to reproduce the error but I added one line to the source code that should fix this bug. Could you install EasyOCR from source code and tell me if it fix the problem?

**Comment by NyxCode** [comment]
@rkcosmos interresting.. Have you tried my snipped with test image I provided?
I tried your fix, and the error does go away, but the prediction for the test image is now
```
[[[[94, 54], [342, 54], [342, 248], [94, 248]], '', 0.0]]
```
with `format="JPEG"`, the prediction is
```
[[[[96, 54], [338, 54], [338, 248], [96, 248]], '3', 0.5106340311409809]]
```

**Comment by rkcosmos** [comment]
solved in version 1.2.5.1
